src/stores/AppStore.js

```diff
@@ -22,7 +22,8 @@ const defaultColorScheme = supportsAutoColorScheme ? ColorScheme.auto : ColorSch
 
 export default {
   state: {
-    imageLoadingStrategy: ImageLoadingStrategy.lazy,
+    imageLoadingStrategy: process.env.VUE_APP_TARGET === 'ide'
```

Comment:
Can you add a test for this?

Reply:
If you want to clean it up a bit you can extract the load strategy to a variable:

```js
const strategy = process.env.VUE_APP_TARGET === 'ide'
      ? ImageLoadingStrategy.eager : ImageLoadingStrategy.lazy
```

Reply:
Discussed with Dobri - we need to make sure to re-import the file during testing, which isn't worth it to me. So I'm gonna keep it this way :)